cocassel/milk-buddy
# View list of containers on the receiver homepage

Once you login as a receiver, you can see a list of containers on the receiver homepage. 

Testing instructions: 
- Login as a transporter and add a couple containers 
- "Collect" milk from farmers for a few containers 
- Click on the "Drop-off" button on the farmer search page
-Login as a transporter 

You should be able to see the list of containers you selected, the amount of milk collected in each container (this will be the container size you selected minus the amount of milk you placed in the container), and the container size
